requirements/local.txt

```diff
@@ -2,81 +2,84 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file local.txt local.in
+#    pip-compile --output-file requirements/local.txt requirements/local.in
 #
-
-alabaster==0.7.8          # via sphinx
+-e git+https://github.com/jazzband/django-debug-toolbar.git@e174168683d9d81e918c2f5e4542079c7606f3ae#egg=django-debug-toolbar==1.6.0.1
```

Comment:
`pip-tools` only supports url based requirements if they have `-e` - tried losing it already :)